docs/pages/Troubleshooting.md

```diff
@@ -23,3 +23,22 @@ rm -rf ~/.ui5/framework/
 ```
 
 Any missing framework dependencies will be downloaded again during the next UI5 Tooling invocation.
+
+## UI5 CLI
```

Comment:
Since the environment variable is most relevant in scenarios where the CLI is **not** used, I'm in favor of using a different headline here.

Reply:
I had in mind to create a separate page for the logger module and describing it there?

Reply:
Please check the BLI. We have agreed on defining it first here and then eventually, with another BLI to create a separate Logger module page

Reply:
> Since the environment variable is most relevant in scenarios where the CLI is **not** used, I'm in favor of using a different headline here.

I got your point but since the `--log-level` is a CLI parameter and the rest of the page refers to the different UI5 repos/projects I put it like that. 
@RandomByte do you have any suggestions on how to name this section?
What do you think about "Environment Variables"? We could eventually extend it in the future with more variables or even consolidate it here.

Reply:
"Environment Variables" sounds good 👌

Reply:
Done